lib/rex/parser/arguments.rb

```diff
@@ -35,6 +52,11 @@ def self.from_s(str)
       # Parses the supplied arguments into a set of options.
       #
       def parse(args, &_block)
+        # e.g. '-x' or '-xyz'
+        short_arg = /^-[a-zA-Z]+$/
+        # e.g. '--verbose', '--very-verbose' or '--very-very-verbose'
+        long_arg = /^--([a-zA-Z]+)((-[a-zA-Z]+)*)$/
```

Comment:
We should probably avoid using `arg` and `flag` interchangeably, as that might confuse the next code reader.
Let's rename these two regexes to use `_flag` :+1: